src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -503,11 +503,21 @@ private BoundAssignmentOperator BindNamedAttributeArgument(AttributeArgumentSynt
         {
             bool wasError;
             LookupResultKind resultKind;
-            Symbol namedArgumentNameSymbol = BindNamedAttributeArgumentName(namedArgument, attributeType, diagnostics, out wasError, out resultKind);
+            Symbol? namedArgumentNameSymbol;
 
-            ReportDiagnosticsIfObsolete(diagnostics, namedArgumentNameSymbol, namedArgument, hasBaseReceiver: false);
+            if (attributeType.IsErrorType())
```

Comment:
Should we just return BadExpression from this code path?

Reply:
Oh, I see what you're saying. We can still bind the RHS value and potentially get a simpler quick exit. Let me try.